templates/00_struct.go.tpl

```diff
@@ -1,10 +1,14 @@
 {{- $alias := .Aliases.Table .Table.Name -}}
+{{- $orig_tbl_name := .Table.Name -}}
 
 // {{$alias.UpSingular}} is an object representing the database table.
 type {{$alias.UpSingular}} struct {
 	{{- range $column := .Table.Columns -}}
 	{{- $colAlias := $alias.Column $column.Name -}}
-	{{if eq $.StructTagCasing "title" -}}
+	{{- $orig_col_name := $column.Name -}}
+	{{if ignore $orig_tbl_name $orig_col_name $.IgnoreFields -}}
+	{{$colAlias}} {{$column.Type}} `{{generateIgnoreTags $.Tags}}boil:"{{$column.Name}}" json:"-" toml:"-" yaml:"-"`
```

Comment:
I don't see a function in this PR named generateIgnoreTags. How does this work?

Reply:
It was already defined, I just reused it:
https://github.com/volatiletech/sqlboiler/blob/5a51e7228cba48f4f40e542504540fcd8b14b26b/templates/00_struct.go.tpl#L17
https://github.com/volatiletech/sqlboiler/blob/5a51e7228cba48f4f40e542504540fcd8b14b26b/strmangle/strmangle.go#L631